lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -1057,12 +1058,21 @@ public void unpublishObjectsExternally(Set<GUID> guids, int accessGroupId) throw
     public List<ObjectData> getObjectsByIds(Set<GUID> ids, PostProcessing pp) 
             throws IOException {
 
-        Map<String, Object> doc = ImmutableMap.of("query",
-                                     ImmutableMap.of("bool",
+        Map<String, Object> query = ImmutableMap.of("bool",
                                         ImmutableMap.of("filter",
                                            ImmutableMap.of("terms",
-                ImmutableMap.of("guid", ids.stream().map(u -> u.toString()).collect(Collectors.toList()))))));
+                ImmutableMap.of("guid", ids.stream().map(u -> u.toString()).collect(Collectors.toList())))));
         //doc.put("_source", Arrays.asList("ojson"));
+        Map<String, Object> highlight =
+                ImmutableMap.of("fields",
+                        ImmutableMap.of("*",
+                                ImmutableMap.of("require_field_match", false)));
+        Map<String, Object> doc = new LinkedHashMap<>();
+        doc.put("query", query);
+
+        if(pp.objectHighlight){
```

Comment:
Space after if and before {

Reply:
is there an auto space detection in eclipse or something? how are you able to spot these spaces/lack of spaces??

Reply:
They stick out like a sore thumb to me